### PR TITLE
allow disabling keep alive

### PR DESCRIPTION
**Pull request by r0fls**
Fixes: https://github.com/channelcat/sanic/issues/637

### CODE REVIEW

File: sanic/server.py [hunk 1]
```diff
@@ -88,10 +89,13 @@ def __init__(self, *, loop, request_handler, error_handler,
         self._timeout_handler = None
         self._last_request_time = None
         self._request_handler_task = None
+        self._no_keep_alive = no_keep_alive
 
     @property
     def keep_alive(self):
-        return self.parser.should_keep_alive() and not self.signal.stopped
+        return (self.parser.should_keep_alive()
```
Comment (messense) [inline sanic/server.py]:
```python
@property
def keep_alive(self):
    return (not self._no_keep_alive
            and not self.signal.stopped
            and self.parser.should_keep_alive())
```

Would be better?

**Comment by 38elements** [comment]
I think it is better to be able to set KEEP_ALIVE by [config](https://github.com/channelcat/sanic/blob/master/sanic/config.py) and there is a problem like #639.

**Comment by r0fls** [comment]
Well this does get loaded from config here: https://github.com/channelcat/sanic/pull/646/files#diff-c4444a35dbd5e37ee480b0e8888e0880R682

How does #639 apply exactly?

**Comment by 38elements** [comment]
I am sorry. 
I was misunderstanding for config.

**Comment by r0fls** [comment]
Do we think the variable makes more sense as just `KEEP_ALIVE` vs `NO_KEEP_ALIVE`?

**Comment by 38elements** [comment]
No, just  I did not read config.py.

**Comment by 38elements** [comment]
About #639, it occurred when I implemented a similar test.

**Comment by 38elements** [comment]
I am sorry for changing my opinion.
Because I feel that `KEEP_ALIVE` is more common, I think that `KEEP_ALIVE` is better.

**Comment by r0fls** [comment]
No worries, I think I agree so better late than never :) I'll update it to use that...

**Comment by 38elements** [comment]
Thanks.

**Comment by 38elements** [comment]
I think that it is  better to add tests for that `keep alive` is valid and invalid.
For #639, I think that the test when `keep alive`  is valid will not work.

**Comment by 38elements** [comment]
I understood the cause.
I commented in #639.

**Comment by 38elements** [comment]
My English is not good.
I am sorry if I confused you.

**Comment by r0fls** [comment]
No you're English is fine. I'm just traveling. I'll try to add the test soon thank you. If you have an idea for it in mind you can put it here otherwise I'll plan to do it soon...

**Comment by 38elements** [comment]
Thanks.

**Pull request merged**